PowerPointLabs/PowerPointLabs/SyncLab/ObjectFormats/ThreeDRotationEffectFormat.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+using System.CodeDom;
+using System.ComponentModel.Design;
+using System.Drawing;
+using System.Windows;
+using Microsoft.Office.Core;
+using PowerPointLabs.ActionFramework.Common.Log;
+using PowerPointLabs.EffectsLab;
+using PowerPointLabs.Utils;
+using Shape = Microsoft.Office.Interop.PowerPoint.Shape;
+using Shapes = Microsoft.Office.Interop.PowerPoint.Shapes;
+using ThreeDFormat = Microsoft.Office.Interop.PowerPoint.ThreeDFormat;
+
+namespace PowerPointLabs.SyncLab.ObjectFormats
+{
+    class ThreeDRotationEffectFormat: Format
+    {
+        private static readonly float TOLERANCE = Single.Epsilon;
```

Comment:
Might be good to add some comments to explain a little about the tolerance

Reply:
It's the common check for floating point numbers, as floating point number checks are never 100% accurate.

Reply:
I've added comments we do the equality check though. Maybe that might make things clearer